examples/custom-universal-routing/next.config.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+  customRoutes: [{
```

Comment:
I like using `next.config.js` as the place for this. But in this file, we might import a lot of server only modules. 
So, when we import this on both environments, that'll be an issue.

So, let's use a different file called `routes.json` or `routes.js`